docs/tests/__snapshots__/props-table.test.ts.snap

```diff
@@ -11556,7 +11556,7 @@ exports[`Props Table 1`] = `
                     \\"BaseTextFieldProps\\": {
                         \\"name\\": \\"BaseTextFieldProps\\",
                         \\"type\\": \\"extends TextFieldOptions, BaseInputProps\\",
-                        \\"description\\": \\"\\",
+                        \\"description\\": \\"0: / 1: * 2: * 3:   4: @ 5: d 6: e 7: p 8: r 9: e 10: c 11: a 12: t 13: e 14: d 15:   16: F 17: o 18: r 19:   20: i 21: n 22: t 23: e 24: r 25: n 26: a 27: l 28:   29: u 30: s 31: e 32:   33: o 34: n 35: l 36: y 37:   38: * 39: /\\",
```

Comment:
This looks like the new jsdoc comment is getting split into an array? Have we verified that this doesn't look weird on the docs site? Do we show these types in props tables?

Reply:
+1. Don't have a ton of understanding on how the props tables work but it does seem like these types should be omitted from it?

Reply:
The script needs to update after `TextArea` is separated from `TextField`. Will update